packages/types/src/base.ts

```diff
@@ -10,6 +10,7 @@ export enum KnownChainIds {
   EthereumMainnet = 'eip155:1',
   AvalancheMainnet = 'eip155:43114',
   BitcoinMainnet = 'bip122:000000000019d6689c085ae165831e93',
+  BitcoincashMainnet = 'bip122:000000000000000000651ef99cb9fcbe',
```

Comment:
To keep the casing consistent? e.g `bitcoincash` when lowercased, `bitcoinCash` when camelcased... `BitcoinCash` when PascalCased? 

```suggestion
  BitcoinCashMainnet = 'bip122:000000000000000000651ef99cb9fcbe',
```

Reply:
updated